package.json

```diff
@@ -86,5 +87,6 @@
       "@commitlint/config-lerna-scopes",
       "@commitlint/config-conventional"
     ]
-  }
+  },
+  "dependencies": {}
```

Comment:
Do we need this ?

Reply:
Yeah, you're right! That's not necessary.

Reply:
Removed!